### ISSUE TITLE
Issue with precision modifier

### ISSUE THREAD

**Issue by eszdman** [LeeTeng2001/better-glsl-plugin#2]
I tried to use this plugin for pixel shaders and compute shaders instead of others buggy on defined functions, but unfortunately got undefined type behaviour
Some screenshot as example
![image](https://user-images.githubusercontent.com/45270494/168495619-e0eff528-8e51-4b9f-8a64-88292253edd9.png)

**Comment by LeeTeng2001** [comment]
Yes I'm sorry about that, currently it doesn't support those keywords (precision, highp), I'll work on it during the next summer break, maybe you can check it out in few weeks?

**Comment by eszdman** [comment]
It should be fine then, also is better to implement define functions/variables highlighting, it's critical for dynamic shader code processing over glsl
